### PR TITLE
ntfsprogs: bump to version 2017.3.23

### PR DESCRIPTION
**Pull request by grueni**
Recompiled with libfuse

The package has no tests.

Tested in VMware with a VM running under ESX 6.5 with updated open-vm-tools und driver/fuse 

```bash
# mount a virtual disk formatted in Windows
# use a writing action
# list the content of the moved folder
# unmount the virtual disk
root@oi-s2:~# ntfs-3g /dev/dsk/c5t2d0p1 /mnt/windows
root@oi-s2:~# mv /mnt/windows/source/test /mnt/windows/
root@oi-s2:~# ls -la /mnt/windows/test/
total 33232
drwxrwxrwx   1 root     root         184 Dec  6 19:36 .
drwxrwxrwx   1 root     root        4096 Dec  6 19:36 ..
-rwxrwxrwx   1 root     root     17010016 Dec  6 19:36 IE8-WindowsXP-x86-DEU.exe
root@oi-s2:~# umount /mnt/windows
```

### CODE REVIEW

File: components/sysutils/ntfsprogs/ntfsprogs.p5m [hunk 1]
```diff
@@ -72,14 +71,10 @@ file path=usr/include/ntfs-3g/types.h
...
 file path=usr/include/ntfs-3g/xattrs.h
-file path=usr/lib/libntfs-3g.a
-link path=usr/lib/libntfs-3g.so target=libntfs-3g.so.871.0.0
-link path=usr/lib/libntfs-3g.so.871 target=libntfs-3g.so.871.0.0
-file path=usr/lib/libntfs-3g.so.871.0.0
+link path=usr/lib/libntfs-3g.so target=libntfs-3g.so.881.0.0
+link path=usr/lib/libntfs-3g.so.881 target=libntfs-3g.so.881.0.0
+file path=usr/lib/libntfs-3g.so.881.0.0
 file path=usr/lib/pkgconfig/libntfs-3g.pc
-file files/fstyp path=usr/lib/fs/ntfs-3g/fstyp
```
Comment (pyhalov) [inline components/sysutils/ntfsprogs/ntfsprogs.p5m:80]: did you miss fstyp ?

Comment (pyhalov) [reply components/sysutils/ntfsprogs/ntfsprogs.p5m:80]: Links here supposed yo be useful.

Comment (grueni) [reply components/sysutils/ntfsprogs/ntfsprogs.p5m:80]: Yes, I overlooked it. But folder fs is history in this version.
Where to put it now?
file files/fstyp path=usr/bin/ntfs-3g.fstyp

Comment (pyhalov) [reply components/sysutils/ntfsprogs/ntfsprogs.p5m:80]: It was never delivered by ntfsprogs, it's illumos interface.

Comment (grueni) [reply components/sysutils/ntfsprogs/ntfsprogs.p5m:80]: Ok. I have to restore the usage of this folder.

**Review by xen0l — APPROVED** [review]

**Pull request merged**